content/api/templates.apib

```diff
@@ -556,26 +402,136 @@ If the template delete API call succeeds, then ALL versions of the template will
+
++ Parameters
+    + draft (optional, boolean, `true`) ...If true, returns the most recent draft template.  If false, returns the most recent published template. When not provided, returns the most recent template version regardless of draft or published.
+    + shared_with_subaccounts (optional, boolean, `true`) ...If true, returns only shared templates. If false, returns only non-shared templates. When not provided, returns both shared and non-shared templates.
+
+
+Returns an array of most recent version of each template in your account.
+Each object returned is an abbreviated version of the full template object with
+the following attributes:
+
+
++ Request
+
+    + Headers
+
+            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+            Accept: application/json
+
++ Response 200 (application/json)
 
         {
-            "errors": [
+            "results": [
                 {
-                    "message": "resource conflict",
-                    "code": "1602",
-                    "description": "Template is in use by msg generation"
+                    "id": "summer_sale",
+                    "name": "Summer Sale!",
+                    "published": true,
+                    "description": "",
+                    "has_draft": true,
+                    "has_published": true,
+                    "last_update_time": "2017-08-11T12:12:12+00:00",
+                    "shared_with_subaccount": true
+                },
+                {
+                    "id": "daily",
+                    "name": "daily",
+                    "published": false,
+                    "description": "Daily roundup email.",
+                    "has_draft": true,
+                    "has_published": true,
+                    "last_use": "2018-05-08T14:49:03+00:00"
+                    "last_update_time": "2018-02-10T14:15:16+00:00",
+                    "shared_with_subaccount": false
                 }
             ]
         }
+
+
+
+### Error Object
+
+On success, the API returns a `results` object along with HTTP 200.
+On failure, an `errors` array will be returned along with HTTP 4xx or 5xx.
+Each error is described in a object with the following fields:
+
++ Data Structure: Attributes
+    + message: (string) - Explains the class of error
+    + code: (string) - Identifies the class of error
+    + description: (string) - Explanation of error
+    + part (enum) - Identifies the MIME part where the error occurred
+        + text
+        + html
+        + Header:Subject
+        + text/plain
+    + line (number) - Identifies the line number within the MIME part identified by the part attribute
++ Sample
+
+    ```json
+    {
+        "message": "substitution language syntax error in template content",
+        "code": "3000",
+        "description": "Error while compiling part text: line 4: syntax error near 'age'",
+        "part": "html",
+        "line": 4,
+    }
+    ```
+
+#### Email RFC 822
```

Comment:
I'd move this after the resource start ("Create an RFC 822 Template") so that it renders correctly.